main.py

```diff
@@ -131,9 +159,13 @@ def process_movie_section(s, watched_set, ratings_dict, listutil, collection):
         except trakt.errors.NotFoundException:
             logging.error(
                 "Movie [{} ({})]: GUID {} not found on trakt".format(movie.title, movie.year, guid))
-        except:
+        except trakt.errors.RateLimitException:
+            logging.warning(
+                "Movie [{} ({})]: Rate Limited. Sleeping 1 sec from trakt (GUID: {})".format(movie.title, movie.year, guid))
+            sleep(1)
+        except Exception as e:
             logging.error(
-                "Movie [{} ({})]: bad response from trakt (GUID: {})".format(movie.title, movie.year, guid))

```

Comment:
This "bad response" string is because sometimes trakt responses are not correct but don't raise a trakt error.
It is raised as a json error by json decoder :
`json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)`

So the e.message is not a response from trakt.

Reply:
I'd like to fix this but my push fails with _permission denied_ from jbalcorn repository.

Reply:
@twolaw I've created new PR based on this PR:
- https://github.com/Taxel/PlexTraktSync/pull/119

you can push and finish up there.